ui/common/src/socket.ts

```diff
@@ -18,11 +18,10 @@ interface MsgOut extends MsgBase {}
 interface MsgAck extends MsgOut {
   at: number;
 }
-type Send = (t: Tpe, d: Payload, o?: any) => void;
```

Comment:
this isn't used anywhere, so the type is described directly where it's used below in Ackable